src/ansible_navigator/__init__.py

```diff
@@ -1,3 +1,2 @@
-""" init
-"""
+"""The initialization file for ansible-navigator."""
```

Comment:
`__init__` represents a package (directory) docstring, not a single module (file). This is why such docstrings are expected to describe the package (as in a Python importable package, not a distribution package).

It'd be something like
```suggestion
"""Ansible-navigator TUI."""
```

You could optionally extend it with "doctest"-style examples or RST code block snippets. For example, look at this https://cheroot.cherrypy.dev/en/latest/pkg/cheroot.cli/ (although this specific example isn't for a package but for a single module, it should demonstrate the idea).

Reply:
@webknjaz in this case the init file is for the ansible_navigator package (the entirety of the source)....  It probably should have been an underscore?

Reply:
got it